src/modules/previewpane/MarkDownPreviewHandler/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("MarkdownPreviewHandler")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("MarkdownPreviewHandler")]
+[assembly: AssemblyCopyright("Copyright ©  2020")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
+// Setting ComVisible to false makes the types in this assembly not visible
+// to COM components.  If you need to access a type in this assembly from
+// COM, set the ComVisible attribute to true on that type.
+[assembly: ComVisible(false)]
+
+// The following GUID is for the ID of the typelib if this project is exposed to COM
+[assembly: Guid("6a71162e-fc4c-4a2c-b90f-3cf94f59a9bb")]
+
+// Version information for an assembly consists of the following four values:
+//
+//      Major Version
+//      Minor Version
+//      Build Number
+//      Revision
+//
+// You can specify all the values or you can default the Build and Revision Numbers
+// by using the '*' as shown below:
+// [assembly: AssemblyVersion("1.0.*")]
+[assembly: AssemblyVersion("1.0.0.0")]
```

Comment:
Todo: Add the assembly similar to other projects (Eg : https://github.com/microsoft/PowerToys/pull/1467)

Reply:
> Todo: Add the assembly similar to other projects (Eg : #1467)

Updated the Assembly Info to be autogenerated for all `c#` projects. So the assembly version will directly pick up from version.props. [ Modified the MSI insatllation](https://github.com/microsoft/PowerToys/blob/25e97f9d3dde751d6f7a66320d37d737d2f62ad5/installer/PowerToysSetup/Product.wxs#L307) to pick up the Assembly Version set in the props file which is required in registry key for .Net Class registration. For MSIX since i'm generating the registry hive(registry.dat) manually from Registry Editor not able to dynamically set the registry key for assembly version. So currently we will need to update the `registry.dat` manually to increment the  assembly version. Possible solutions can be programmatically generate the `registry.dat` file but I didn't find a way to generate `registry.dat` [without modifying the registry](https://helgeklein.com/blog/2009/08/registry-fun-working-with-hive-files/) and then import the hive or explore alternative to use Mixed Assembly as pointed by @yuyoyuppe in comment to avoid using the `registry.dat`

@crutkas / @ryanbodrug-microsoft What is the priority of this. I think this can be addressed in a separate PR or Issue. Let me know what you think?

Reply:
> but I didn't find a way to generate registry.dat without modifying the registry and then import the hive or explore alternative

I think it's ok to modify the registry and then generate `registry.dat` from it, since we'll only be doing that on the isolated CI server. It shouldn't break the ability to iterate locally on the MSIX versionf of Preview handler, right?

Reply:
> I think it's ok to modify the registry and then generate `registry.dat` from it, since we'll only be doing that on the isolated CI server.

Yeah it should be fine to generate on isolated CI server. I'm not sure if we will have access to the registry of the CI Server.

>  It shouldn't break the ability to iterate locally on the MSIX versionf of Preview handler, right?

To iterate locally on the MSIX version we will need to update assembly version in `registry.dat` either with the programmatically approach or manually mount the hive on a empty key in registry and update the key with registry editor and export back.

Reply:
Created a issue for this https://github.com/microsoft/PowerToys/issues/1545